StatusList.js

```diff
@@ -1,22 +1,22 @@
 /*!
- * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
  */
 import Bitstring from '@digitalbazaar/bitstring';
 
-export default class RevocationList {
+export class StatusList {
   constructor({length, buffer} = {}) {
     this.bitstring = new Bitstring({length, buffer});
     this.length = this.bitstring.length;
   }
 
-  setRevoked(index, revoked) {
-    if(typeof revoked !== 'boolean') {
-      throw new TypeError('"revoked" must be a boolean.');
+  setStatus(index, revokedStatus) {
```

Comment:
This needs to be renamed to be more generic to support statuses other than "revocation" (such as "suspension").

```suggestion
  setStatus(index, status) {
```